axum-extra/Cargo.toml

```diff
@@ -32,7 +32,7 @@ axum-macros = { path = "../axum-macros", version = "0.2", optional = true }
 serde = { version = "1.0", optional = true }
 serde_json = { version = "1.0.71", optional = true }
 percent-encoding = { version = "2.1", optional = true }
-cookie-lib = { package = "cookie", version = "0.16", features = ["percent-encode", "signed"], optional = true }
+cookie-lib = { package = "cookie", version = "0.16", features = ["percent-encode", "signed", "private"], optional = true }
```

Comment:
Maybe pass through the features instead? I.e. have a `signed-cookies` feature that activates `SignedCookieJar` and `cookie-lib/signed`, equivalent for `private` / `PrivateCookieJar`.

Reply:
I played around with some time ago but couldn't find a decent way of passing in the `Key`. You can get it manually via an extension and pass it in when passing `.private_jar(key)` but didn't like that too much 🤷

Reply:
and yeah did think about adding separate features for enabling signed or private jars since it pulls in a few deps but thats a breaking change and we just shipped a new major yesterday :P

Reply:
yeah thats true :P